issue found on: server/server.go:53:2: importShadow

an issue has been detected in source code: ./server/server.go:53:2: importShadow: shadow of imported from 'github.com/RedHatInsights/insights-results-aggregator-mock/storage' package 'storage'

--- a/server/server.go
+++ b/server/server.go
@@ -50,12 +50,12 @@ type HTTPServer struct {
 
 // New constructs new implementation of Server interface
 func New(config Configuration,
-	storage storage.Storage,
+	storageInstance storage.Storage,
 	groups map[string]groups.Group,
 	content []content.RuleContent) *HTTPServer {
 	return &HTTPServer{
 		Config:  config,
-		Storage: storage,
+		Storage: storageInstance,
 		Groups:  groups,
 		Content: content,
 	}

